ED2/ensembles_modeling/predict_growth.Rmd

```diff
@@ -265,17 +270,17 @@ accuracy vs interpretability? former
 for predicting, give it the same input data but change the quantum efficiency parameter? 
 
 
-### Random forest
 
-split dataset
+### subset dataset
+
 ```{r}
 set.seed(1234)
 ens_params_env %>% group_by(site, year) %>% summarise(n = n())
 
 t <- ens_params_env %>% 
   filter(site == "WL" & year > 2012) %>% 
```

Comment:
What is the reasoning behind including only this one site, and these years?

Reply:
that was a mistake, meant to _remove_ these runs because there were too few ensemble members

```suggestion
  filter(!(site == "WL" & year > 2012)) %>% 
```